staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/envelope.go

```diff
@@ -51,7 +51,7 @@ func init() {
 
 const (
 	// KMSAPIVersion is the version of the KMS API.
-	KMSAPIVersion = "v2beta1"
+	KMSAPIVersion = "v2"
```

Comment:
We have to support both (likely forever because there is no good reason to break clients).